src/tools/cephfs/top/cephfs-top

```diff
@@ -292,7 +296,14 @@ if __name__ == '__main__':
                         help='Path to cluster configuration file')
     parser.add_argument('--selftest', dest='selftest', action='store_true',
                         help='run in selftest mode')
+    parser.add_argument('-d','--delay', nargs='?', default=REFRESH_INTERVAL,
+                        type=float,help='Interval to refresh data (defualt: 1)')
+
     args = parser.parse_args()
+    if args.delay < MIN_REFRESH_INTERVAL:
+         parser.print_help()
```

Comment:
nit: Is there a need to print full help? With @tchaikov `type` suggestion, this might not be required.